data/match/src/commonMain/kotlin/com/adammcneilly/pocketleague/data/match/GetPastWeeksMatchesUseCase.kt

```diff
@@ -0,0 +1,29 @@
+package com.adammcneilly.pocketleague.data.match
+
+import com.adammcneilly.pocketleague.core.models.Match
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import kotlinx.datetime.Clock
+import kotlin.time.Duration.Companion.days
+
+/**
+ * Return an observable type of matches for the past week.
+ */
+class GetPastWeeksMatchesUseCase(
```

Comment:
I think we can move this into shared/app because then we can also have it map to a display model. 👀